src/main/kotlin/frc/robot/Constants.kt

```diff
@@ -13,6 +22,36 @@ private val SPEAKER_POSE_BLUE = Translation2d(0.0, 5.5479442)
 val SPEAKER_POSE: Translation2d
     get() = SPEAKER_POSE_BLUE.flipIfNeeded()
 
+private fun getValueFromJson(element: JsonElement, valName: String): Double =
+    element.jsonObject[valName]?.jsonObject?.get("val")?.jsonPrimitive?.doubleOrNull ?: 0.0
+
+private fun parseChoreoPoses(): Map<String, Pose2d> {
+    val json = Json { ignoreUnknownKeys = true }
+    val jsonObject =
+        json
+            .parseToJsonElement(
+                File(
+                        "${Filesystem.getDeployDirectory()}/choreo/autotrajectories/AutoPaths.chor"
+                    )
+                    .readText()
+            )
+            .jsonObject
+
+    val jsonPoses =
+        jsonObject["variables"]?.jsonObject?.get("poses")?.jsonObject
+            ?: return emptyMap()
+
+    return jsonPoses.mapValues { (key, pose) ->
+        Pose2d(
+            getValueFromJson(pose, "x"),
+            getValueFromJson(pose, "y"),
+            Rotation2d(getValueFromJson(pose, "heading"))
+            )
+    }
+}
+
+val choreoPoses = parseChoreoPoses()
```

Comment:
```suggestion
val ALIGNMENT_POSES = parseChoreoPoses()
```
1. It's a constant. 
2. The fact it came from Choreo is an implementation detail.

Also, perhaps all this code should be moved to an autos directory? I think it will be convenient to have an autos directory, with sub-files or sub-directories for the auto period and for auto alignment.

Reply:
d74c898240c5090af47fa0387b7799900b9b1e7f

Reply:
Great. What's your opinion on the second thing?

Reply:
Oh I missed it, yeah I think it's a good idea' I'll do it

Reply:
96d34a7beaf25a17f5a765d9c5d13de174831926